File: wrench-app/src/main/java/com/izettle/wrench/database/tables/ConfigurationValueTable.java
package com.izettle.wrench.database.tables;

import com.izettle.wrench.core.ColumnNames;

public interface ConfigurationValueTable {
    String TABLE_NAME = "configurationValue";
    String COL_ID = ColumnNames.Bolt.COL_ID;
    String COL_CONFIG_ID = "configurationId";
    String COL_VALUE = ColumnNames.Bolt.COL_VALUE;
    String COL_SCOPE = "scope";
}

File: wrench-app/src/main/java/com/izettle/wrench/database/tables/ConfigurationValueTable.kt
package com.izettle.wrench.database.tables

import com.izettle.wrench.core.ColumnNames

interface ConfigurationValueTable {
    companion object {
        const val TABLE_NAME = "configurationValue"
        const val COL_ID = ColumnNames.Bolt.COL_ID
        const val COL_CONFIG_ID = "configurationId"
        const val COL_VALUE = ColumnNames.Bolt.COL_VALUE
        const val COL_SCOPE = "scope"
    }
}
